src/main/java/com/google/sps/classes/utilityClass.java

```diff
@@ -0,0 +1,31 @@
+ 
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+import com.google.gson.Gson;
+
+/**
+ * Utility methods used across classes. Just import class to access all methods.
+ */
+public final class UtilityClass {
+  /**
+   * Converts objects to JSON using GSON class.
+   */
+  public static String convertToJsonUsingGson(Object object) {
```

Comment:
A good practice for any java code you write is to include junit tests for those methods. This could be a very simple test, just showing that valid and expected JSON is returned for some object.

Up to you whether you want to do this now or in a fast follow-up PR (theoretically you don't need any tests but I'm going to push for them, especially for your utility Java functions). If you decide to follow up on this, please include a TODO!

Reply:
Will do!